lib/fetch.js

```diff
@@ -55,6 +54,32 @@ export function mapAndSanitate(item, type) {
   };
 }
 
+export function sanitize(key, type, defaultValue, json) {
```

Comment:
> What's the convention for arg order? In non-curried languages I like (haystack, needle, options)
Not sure there's a convention per se. I think a data-last arg order is fine - will make it easier if later down the line, we make that method curried.